git-quick-stats.1

```diff
@@ -7,6 +7,8 @@
 For non\-interactive mode:
 .B git\-quick\-stats [OPTIONS]
 .PP
+.B git\-quick\-stats\-csv [OPTIONS]
+.PP
```

Comment:
This portion doesn't exactly make sense. For example, if we were to type `git-quick-stats-csv`, this would not execute. Let's get rid of this entirely.